<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VLCync</class>
 <widget class="QMainWindow" name="VLCync">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>282</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>VLCync</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#121212</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>281</width>
      <height>350</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="loginScreen">
     <widget class="QLabel" name="label_serverIP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="baseSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Mono</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
      <property name="text">
       <string>Server IP:Port -</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_ip">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
     </widget>
     <widget class="QPushButton" name="connectButton">
      <property name="geometry">
       <rect>
        <x>91</x>
        <y>280</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Mono</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd;background-color:#484848</string>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_username">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Mono</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
      <property name="text">
       <string>Enter a username -</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_username">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_output">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>241</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Noto Mono</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inp_password">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#ddd</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lobbyScreen">
     <widget class="QPushButton" name="mediaSelectButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#484848; color:#ddd</string>
      </property>
      <property name="text">
       <string>Select media</string>
      </property>
     </widget>
     <widget class="QLabel" name="fileNameDisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>250</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#484848; color:#ddd</string>
      </property>
      <property name="text">
       <string>No file selected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="voteButton">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>280</y>
        <width>120</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#484848; color:#ddd</string>
      </property>
      <property name="text">
       <string>Vote to start</string>
      </property>
     </widget>
     <widget class="QPushButton" name="disconnectButton">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>160</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#484848; color:#ddd</string>
      </property>
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>260</width>
        <height>140</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>240</width>
            <height>120</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="liveConsoleOutput">
             <property name="font">
              <font>
               <pointsize>9</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:#484848; color:#ddd</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
